lib/frameworkhelper.js

```diff
@@ -21,15 +23,14 @@ function flog(msg) {
 }
 
 function cloneFramework (next) {
-  var frameworkPath = FRAMEWORK_DIR;
-  fs.exists (frameworkPath, function (exists) {
+  fs.exists (path.join(process.cwd(), FRAMEWORK_DIR), function (exists) {
```

Comment:
The serverRoot property is available as soon as the config module is required in - see https://github.com/adaptlearning/adapt_authoring/blob/develop/lib/configuration.js#L65

(This property needs to be available as early as possible, as much of the startup process depends on it)